.idea/gradle.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is already in DEV. Check the difference between DEV and this branch and you will see.